appservice/src/main/java/nu/yona/server/device/rest/DeviceController.java

```diff
@@ -397,18 +460,30 @@ public DeviceResource toResource(DeviceBaseDto device)
 			ControllerLinkBuilder selfLinkBuilder = getSelfLinkBuilder(device.getId());
 			addSelfLink(selfLinkBuilder, deviceResource);
 			addEditLink(selfLinkBuilder, deviceResource);
+			if (isRequestingDevice(device))
+			{
+				addPostOpenAppEventLink(deviceResource);
+				addAppActivityLink(deviceResource);
+				addSslRootCertificateLink(deviceResource);
```

Comment:
❓  This link is actually not different per device, is it? So why move it here?

Reply:
Though it's not specific for the device, it's really device information. I want to group all such information on the device resource.